### PR TITLE
luhn: add default param

### LINKED ISSUE #571: luhn: add parameters to exercise placeholder
We would like you to help us add expected parameters in placeholder templates as described in #509 to make it easier for learners to start working on the exercise in question.

Feel free to ask any questions, we'll be glad to help!

### PR DESCRIPTION
**Pull request by denvaar**
Fixes #571

### CODE REVIEW

File: exercises/luhn/luhn.py [hunk 1]
```diff
@@ -1,3 +1,3 @@
 class Luhn(object):
-    def __init__(self):
+    def __init__(self, string):
```
Comment (m-a-ge) [inline exercises/luhn/luhn.py]: This is too general. 
Look at https://github.com/exercism/python/tree/master/exercises/luhn it requires not just a string, it requires string of numbers.
Could you propose a better name?

Comment (jnaulty) [reply exercises/luhn/luhn.py]: it does not requires a string of numbers, it validates a string of numbers.
From the luhn/README.md: 
```
The Luhn algorithm is a simple checksum formula used to validate a variety of identification numbers,
```

Comment (m-a-ge) [reply exercises/luhn/luhn.py]: @jnaulty I see, maybe I was not clear enough in my comment. 
My vision is to provide good parameter names to the learner, as clearer as possible.
What about `string_with_numbers`?

Concerning test cases, there is a special repository for defining exercises - https://github.com/exercism/problem-specifications/. For this exercise it is https://github.com/exercism/problem-specifications/blob/master/exercises/luhn/canonical-data.json. Not sure if it's a good idea to provide a test case for non string input, even though I personally always add this kind of verifications

**Comment by N-Parsons** [comment]
@jnaulty, are you still working on this?

**Comment by jnaulty** [comment]
@N-Parsons I was actually just commenting on this...
this PR is from @denvaar

**Comment by N-Parsons** [comment]
Sorry, I tagged the wrong person!

@denvaar, are you still working on this?

**Pull request closed without merging**